specification/compute/resource-manager/Microsoft.Compute/2017-03-30/compute.json

```diff
@@ -3413,11 +3413,11 @@
         },
         "type": {
           "type": "string",
```

Comment:
based on the description you may want to evaluate turning this property into an enum in the future - https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-enum